sklearn/ensemble/tests/test_forest.py

```diff
@@ -1377,3 +1377,4 @@ def test_little_tree_with_small_max_samples(ForestClass):
 
     msg = "Tree without `max_samples` restriction should have more nodes"
     assert tree1.node_count > tree2.node_count, msg
+
```

Comment:
This is the PEP8 error. You can remove it and it will be fine.

Reply:
More precisely, this causes the following error in the "linting" steps in the Continuous Integration report below:

```
Running flake8 on the diff in the range c9f8f52a7..0ecf35612 (9 commit(s)):
--------------------------------------------------------------------------------
sklearn/ensemble/tests/test_forest.py:1380:1: W391 blank line at end of file
```